astropy/coordinates/builtin_frames/altaz.py

```diff
@@ -45,18 +45,18 @@
         The location on the Earth.  This can be specified either as an
         `~astropy.coordinates.EarthLocation` object or as anything that can be
         transformed to an `~astropy.coordinates.ITRS` frame.
-    pressure : `~astropy.units.Quantity`
+    pressure : `~astropy.units.Quantity` ['pressure']
         The atmospheric pressure as an `~astropy.units.Quantity` with pressure
         units.  This is necessary for performing refraction corrections.
         Setting this to 0 (the default) will disable refraction calculations
         when transforming to/from this frame.
-    temperature : `~astropy.units.Quantity`
+    temperature : `~astropy.units.Quantity` ['temperature']
         The ground-level temperature as an `~astropy.units.Quantity` in
         deg C.  This is necessary for performing refraction corrections.
-    relative_humidity : `~astropy.units.Quantity` or number.
+    relative_humidity : `~astropy.units.Quantity` or number
```

Comment:
Can we put ['dimensionless'] or similar here?

Reply:
Done!